Projects/DbCLI/CreateDb.py

```diff
@@ -439,7 +440,8 @@ def CreateDb(options, dataFilename='', supplier=None):
   if options.loadMols:
     if len(options) != 1:
```

Comment:
You need to change this test as well. What is tested here is if the user provided the `filename` or not. If `loadMols` is set, then the user must provide it.

Reply:
Yes, this will also follow the same, `options.filename`.